cmd/account/rotate-secret.go

```diff
@@ -208,8 +215,12 @@ func (o *rotateSecretOptions) run() error {
 	}
 
 	// Update osdManagedAdmin secrets
-	// Username is osdManagedAdmin-aaabbb
-	osdManagedAdminUsername := common.OSDManagedAdminIAM + "-" + accountIDSuffixLabel
+	var osdManagedAdminUsername string
+	if o.osdManagedAdminUsername != "" {
+		osdManagedAdminUsername = o.osdManagedAdminUsername
+	} else {
+		osdManagedAdminUsername = common.OSDManagedAdminIAM + "-" + accountIDSuffixLabel
+	}
```

Comment:
I feel like this could be simplified and get rid of the else... something like the following probably?
```suggestion
	osdManagedAdminUsername := o.osdManagedAdminUsername
	if osdManagedAdminUsername == "" {
		osdManagedAdminUsername = common.OSDManagedAdminIAM + "-" + accountIDSuffixLabel
	}
```